policy/modules/kernel/devices.if

```diff
@@ -1937,6 +1937,25 @@ interface(`dev_rw_dlm_control',`
 	rw_chr_files_pattern($1, device_t, dlm_control_device_t)
 ')
 
+########################################
+## <summary>
+##	Automatic type transition to the type
+##	for the console device node when created in /dev.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`dev_filetrans_console_dev',`
+	gen_require(`
+		type device_t;
+	')
+
+	term_create_console($1, device_t)
```

Comment:
how about: `dev_filetrans($1, console_device_t, chr_file, "console")` instead?

Reply:
I did it this way to avoid requiring the `console_device_t` type which belongs to the `terminal` module instead, though I much prefer keeping all the required access in this interface if possible.